recipes/pyTMD/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
This is included in the PyPI tarball so you shouldn't include it here.

Reply:
sounds good :+1: